pmd-java/src/test/java/net/sourceforge/pmd/lang/java/ast/AccessNodeTest.java

```diff
@@ -139,9 +139,9 @@ public void testPackagePrivate() {
 
 
     private static String makeAccessJavaCode(String[] access, String declRest) {
-        String result = "public class Test { ";
+        StringBuilder result = new StringBuilder("public class Test { ");
```

Comment:
Please don't edit this file as it has been removed in another branch

Reply:
@oowekyala reverted&rebased